.github/workflows/release.yml

```diff
@@ -1,73 +1,57 @@
-name: Release
+name: Version 🔖
 
-# Releases a canary release when new commits merge to main
-# Releases an official release when workflow dispactch is
-# manually triggered
 on:
-  workflow_dispatch:
   push:
     branches:
-      - main
+      - v1
```

Comment:
This should not be v1 branch.  Do we suspect we will be making any more prs to v1 branch?  If so we can just use the old publishing workflow for v1

Reply:
don't suspect to be making any more PRs to the v1 branch!